tox.ini

```diff
@@ -123,6 +123,7 @@ commands =
   bash -c "git ls-files docs/*.rst | while read file; do if ! grep -q :description: $file; then echo E: no description meta tag in $file; exit 1; fi; done"
   bash -c "cd docs && yarn && yarn build-bootstrap"
   sphinx-build -W -b dirhtml docs/source docs/build
+  sphinx-build -W -b spelling docs/source docs/build
```

Comment:
I'd do that first so we don't build the html doc if there are spelling mistakes.
It's a small optimization but it's a nice way to save some CI time. :)